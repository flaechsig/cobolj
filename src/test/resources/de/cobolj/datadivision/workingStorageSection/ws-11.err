Kombinierter Test mit mÃ¶glichen Typen