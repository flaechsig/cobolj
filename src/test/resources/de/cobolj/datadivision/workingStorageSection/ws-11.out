WS-NUM1 : +000.00
WS-NUM3 : -123.45
WS-NAME : ABCDEF
WS-ID : A121$
